store/src/java/com/zimbra/cs/service/PreAuthServlet.java

```diff
@@ -248,6 +248,7 @@ public void doGet(HttpServletRequest req, HttpServletResponse resp)
                     else
                         at = (expires ==  0) ? AuthProvider.getAuthToken(acct) : AuthProvider.getAuthToken(acct, expires);
 
+                    at.setCsrfTokenEnabled(true);
```

Comment:
we are generating auth token on line no 145 as well, don't we need this fix there as well?

Reply:
@silentsakky That authtoken is generated by auth token passed in request while doing Preauth so will keep all the original properties. So no need to handle here.